src/planning/tests/suggestion-composer-test.ts

```diff
@@ -9,59 +9,45 @@
 import {storageKeyPrefixForTest} from '../../runtime/testing/handle-for-test.js';
-import {StubLoader} from '../../runtime/testing/stub-loader.js';
+import {Loader} from '../../platform/loader.js';
 import {TestVolatileMemoryProvider} from '../../runtime/testing/test-volatile-memory-provider.js';
-import {HeadlessSuggestDomConsumer} from '../headless-suggest-dom-consumer.js';
-import {PlanningModalityHandler} from '../planning-modality-handler.js';
 import {Planner} from '../planner.js';
 import {Speculator} from '../speculator.js';
-import {SuggestionComposer} from '../suggestion-composer.js';
-import {ConCap} from '../../testing/test-util.js';
+//import {SuggestionComposer} from '../suggestion-composer.js';
 import {StrategyTestHelper} from '../testing/strategy-test-helper.js';
 
-class TestSuggestionComposer extends SuggestionComposer {
-  get suggestConsumers() {
-    return this._suggestConsumers;
-  }
-}
-
-class TestSlotComposer extends MockSlotComposer {
-  constructor(options: SlotComposerOptions = {}) {
-    super({
-        strict: false,
-        modalityHandler: PlanningModalityHandler.createHeadlessHandler(), ...options
-    });
-  }
-}
+// class TestSuggestionComposer extends SuggestionComposer {
+//   get suggestConsumers() {
+//     return this._suggestConsumers;
+//   }
+// }
 
 describe('suggestion composer', () => {
   it('singleton suggestion slots', async () => {
-    const loader = new StubLoader({});
+    const loader = new Loader();
     const memoryProvider = new TestVolatileMemoryProvider();
     const context = await Manifest.load('./src/runtime/tests/artifacts/suggestions/Cake.recipes', loader, {memoryProvider});
-    const runtime = new Runtime({
-        loader, composerClass: TestSlotComposer, context, memoryProvider});
+    const runtime = new Runtime({loader, context, memoryProvider});
     const arc = runtime.newArc('demo', storageKeyPrefixForTest());
 
-    const slotComposer = arc.pec.slotComposer as MockSlotComposer;
-    slotComposer.newExpectations('debug');
+    const slotComposer = arc.pec.slotComposer;
+    const observer = new SlotTestObserver();
+    slotComposer.observeSlots(observer);
 
     const suggestions = await StrategyTestHelper.planForArc(arc);
-
-    const suggestionComposer = ConCap.silence(() => new TestSuggestionComposer(arc, slotComposer));
-    await suggestionComposer.setSuggestions(suggestions);
     assert.lengthOf(suggestions, 1);
-    assert.isEmpty(suggestionComposer.suggestConsumers);
 
-    // TODO(sjmiles): uses old render data, will be repaired in subsequent PR
-    //slotComposer.newExpectations()
-    //    .expectRenderSlot('MakeCake', 'item', {'contentTypes': ['template', 'model', 'templateName']});
+    //const suggestionComposer = ConCap.silence(() => new TestSuggestionComposer(arc, slotComposer));
+    //await suggestionComposer.setSuggestions(suggestions);
+    //assert.isEmpty(suggestionComposer.suggestConsumers);
+
+    observer.newExpectations()
```

Comment:
nit - same line might be more readable

Reply:
I guess my preference is to err on the side of consistency, and do all of these as multiline constructions. It makes it easy to add remove expectations.